/* ==========================================================
 *
 * Generated by system . DO NOT EDIT!
 * ========================================================== */


// gsk_pb_report_data;
message pb_report_t_base_info
{
    optional uint32               f_uin                           = 1;  //--用户ID
    optional int32                f_clt                           = 2;  //--客户端id(ios/android/pc/mac)
    optional string               f_phone                         = 3;  //--手机号
    optional int32                f_dye                           = 4;  //--染色ID
    optional string               f_ip                            = 5;  //--ip
    optional string               f_name                          = 6;  //--用户昵称
    optional string               f_province                      = 7;  //--省
    optional string               f_city                          = 8;  //--市
    optional string               f_company_type                  = 9;  //--公司类型
    optional string               f_years_of_working              = 10;  //--工作年限
    optional string               f_job_type                      = 11;  //--工作类型
    optional string               f_job_title                     = 12;  //--职称
}

//-----------------------------上报
message pb_report_t_client
{
    optional string               f_phone_info                    = 1;  //--手机信息
    optional string               f_os                            = 2;  //--手机操作系统
    optional string               f_sp                            = 3;  //--手机运营商
    optional string               f_network                       = 4;  //--手机网络
}

message pb_report_t_report
{
    optional int32                f_msg_type                      = 1;  //--1、请求
    optional int32                f_i_cmd                         = 2;  //--命令字
    optional string               f_s_cmd                         = 3;  //--命令字描述
    optional int32                f_time                          = 4;  //
    optional string               f_log                           = 5;  //
    optional pb_report_t_base_info f_info                          = 6;  //--用户信息
    optional pb_report_t_client   f_client                        = 7;  //--客户端上报信息
}

/* ==========================================================
 *
 * Generated by system . DO NOT EDIT!
 * ========================================================== */


